cookbooks/fb_systemd/recipes/udevd.rb

```diff
@@ -18,19 +18,26 @@
 # limitations under the License.
 #
 
+udevadm = value_for_platform(
+  'centos' => {
+    '< 6.0' => '/sbin/udevadm'
```

Comment:
TIL operators are a thing. Looks like this goes back to at least Chef 12 (https://docs-archive.chef.io/release/12-0/dsl_recipe.html#id4) so it should be safe to use.